index.js

```diff
@@ -64,12 +64,15 @@ function get(url, timeout, port, protocol, detailed) {
   protocol = protocol || 'https:';
 
   var options = getOptions(url, port, protocol);
+  
+  if (timeout)
```

Comment:
Many JS style guides prohibit `if` without a block, always requiring braces even for a single line. How do you feel about that?